cmake/projects/gst_plugins_ugly/hunter.cmake

```diff
@@ -40,6 +51,7 @@ hunter_download(
     "lib/gstreamer-1.0/libgstasf.la"
     "lib/gstreamer-1.0/libgstdvdlpcmdec.la"
     "lib/gstreamer-1.0/libgstdvdsub.la"
-    "lib/gstreamer-1.0/libgstrmdemux.la"
```

Comment:
Since we are keeping the old version in `hunter.cmake` (user can switch to the old version in local config) we can't just remove this. I think the right approach here will be to check the version, see:
- https://github.com/ruslo/hunter/blob/v0.23.193/cmake/projects/xcb/hunter.cmake#L50-L52

Reply:
Ok, I'm working on that. By the way, I added the x264 dependency for the ugly plugin, but it is not needed. should I add an option, for example GST_PLUGIN_BASE_WITH_H264 ->add the dependency? If so is there another project I can use as a template?

Reply:
> should I add an option, for example GST_PLUGIN_BASE_WITH_H264 ->add the dependency?

It's already in `CMAKE_ARGS`, user can specify custom `DEPENDS_ON_PACKAGES` without `x264`. But probably it will affect `PACKAGE_UNRELOCATABLE_TEXT_FILES` so it will not work in practice. Workaround will be to use optional `PACKAGE_UNRELOCATABLE_TEXT_FILES`: https://github.com/ruslo/hunter/pull/1864

Custom schemes are hairy, as usual.

Reply:
But can I mix PACKAGE_UNRELOCATABLE_TEXT_FILES and PACKAGE_UNRELOCATABLE_TEXT_FILES_OPTIONAL? I tried, but I've got an error that "PACKAGE_UNRELOCATABLE_TEXT_FILES_OPTIONAL file" is not found...

Reply:
Yes, it is not working, if compiled without the x264 dependency it complains about the missing file. PACKAGE_UNRELOCATABLE_TEXT_FILE_OPTIONALS is still a PR, what would you like me to do now?

Reply:
> what would you like me to do now?

I'm okay with defaults if both new and old versions are working.

Reply:
> I'm okay with defaults if both new and old versions are working.

I've added `gst_plugins_good-1.10` example for testing.